Server/src/routes/fileUploadRouter.ts

```diff
@@ -16,7 +16,7 @@ let router = Router();
  * and the file is stored in a temporary directory called tmp/json. This route is referred for processing by 
  * the service.
  */
-router.post('/dataupload', [JWTAuthenticator.verifyJWT, upload.single('file')], (request: MulterRequest, response: Response, next: NextFunction) => {
+router.post('/dataupload', upload.single('file'), (request: MulterRequest, response: Response, next: NextFunction) => {
```

Comment:
This change cant be commited - has to keep JWTAuthenticator.verifyJWT...

Reply:
Had to remove that line before because the file upload would not work when its there. its now re added on the recent commit.